src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF1008.PhysicalLocationPropertiesMustBeConsistent_Invalid.sarif

```diff
@@ -28,6 +28,154 @@
               }
             }
           ]
+        },
+        {
+          "ruleId": "TST0002",
+          "level": "error",
+          "message": {
+            "text": "Bad result - partially overlapping region and contextregion (line/column based)."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src/test.c"
+                },
+                "region": {
+                  "startLine": 1,
+                  "startColumn": 5,
+                  "endLine": 10,
+                  "endColumn": 5
+                },
+                "contextRegion": {
+                  "startLine": 1,
+                  "startColumn": 1,
+                  "endLine": 10,
+                  "endColumn":15
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "TST0003",
+          "level": "error",
+          "message": {
+            "text": "Bad result - non-overlapping region and contextregion (line/column based)."
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src/test.c"
+                },
+                "region": {
+                  "startLine": 5,
+                  "endLine": 10
+                },
+                "contextRegion": {
+                  "startLine": 2,
+                  "endLine": 4
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "TST0004",
+          "level": "error",
+          "message": {
+            "text": "Bad result - fully overlapping region and contextregion (line/column based)."
```

Comment:
>fully overlapping [](start = 34, length = 17)

Let's say "identical" -- I think it better conveys that we've violated the _proper_ superset requirement.